wazo_acceptance/helpers/asterisk.py

```diff
@@ -10,5 +10,11 @@ def __init__(self, ssh_client):
     def send_to_asterisk_cli(self, asterisk_command):
         return self._ssh_client.out_call(self._format_command(asterisk_command))
 
+    def send_chan_test_command(self, command):
```

Comment:
How about detecting chan_test commands (if they start with `test ` for example)?

I know that I have more chances to use `send_to_asterisk_cli` than `send_chan_test_command`.